.env.dist

```diff
@@ -64,8 +55,8 @@ REDIS_HOST=redis
 REDIS_PORT=6379
 REDIS_AUTH=redis_pass
 
-MAILER_EMAIL=info@example.com
-MAILER_FROM=CMP
+MAILER_EMAIL=info@cmp.io
```

Comment:
The `example.com` will be more suitable since the `cmp.io` is not our domain and and may confuse the users.